lib/DDGC/Web/Controller/InstantAnswer.pm

```diff
@@ -945,7 +948,7 @@ sub asana :Chained('base') :PathPart('asana') :Args(0) {
         $result = {};
     };
 
-    $c->stash->{x}->{result} = $result->decoded_content;
+    $c->stash->{x}->{result} = $result;
```

Comment:
Seems like no one is really using that functionality, you know.
 I tested it today while enforcing action_token stuff and... :)